apis/Google.Cloud.DevTools.ContainerAnalysis.V1/Google.Cloud.DevTools.ContainerAnalysis.V1.SmokeTests/ContainerAnalysisSmokeTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Cloud.DevTools.ContainerAnalysis.V1.SmokeTests
+{
+    using Google.Api.Gax;
+    using Grafeas.V1;
+    using System;
+
+    public class ContainerAnalysisSmokeTest
+    {
+        public static int Main(string[] args)
+        {
+            // Create client
+            ContainerAnalysisClient containerAnalysisClient = ContainerAnalysisClient.Create();
+            // ContainerAnalysisClient doesn't contain Grafeas methods; only IAM, and a way of getting at a GrafeasClient.
+            GrafeasClient grafeasClient = containerAnalysisClient.GrafeasClient;
+
+            // Call API method
+            ProjectName projectName = new ProjectName(args[0]);
```

Comment:
Might be worth showing a useful error if an arg is not given?

Reply:
Ah yes, we do that in others:
https://github.com/googleapis/google-cloud-dotnet/blob/master/apis/Google.Cloud.Trace.V2/Google.Cloud.Trace.V2.SmokeTests/TraceServiceSmokeTest.g.cs

Will add to both smoke tests.